db/seeds.rb

```diff
@@ -1,7 +1,47 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+#Companies list
+#name, ticker, shares, starting_price
+companies_list = [
+  [ "Apple", 	"AAPL", 	20, 200],
+  [ "Google", 	"GOOGL", 	30, 110],
+  [ "Facebook", "FB", 		15, 100],
+  [ "Amazon", 	"AMZN", 	20, 150]
+]
+
+companies_list.each do |name, ticker, shares, price|
+  #Crate companies entries
+  company = Company.create(
+		  	name:		name, 
+		  	ticker:		ticker,  
+		  	shares:		shares
+	  	)
+
+  #Create CEO users entries
+  ceo = User.create(
+  			name: 		name + "-CEO", 
+  			balance: 	0
```

Comment:
I'd kinda like to use the CEOs as "rich users that will always exist" in the Tsung tests.  Can we start them out with a million dollars or something?